vcpkg/maintainers/functions/vcpkg_make_install.md

```diff
@@ -0,0 +1,103 @@
+---
+title: vcpkg_make_install
+description: Use vcpkg_make_install to build and install a Make-based project.
+ms.date: 01/31/2024
+---
+# vcpkg_make_install
+
+Build and install a Make-based project.
+
+## Usage
+
+```cmake
+vcpkg_make_install(
+    [DISABLE_PARALLEL]
+    [NO_DESTDIR]
+    [NO_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [LOGFILE_ROOT <root-name>]
+    [SUBPATH <sub-directory>]
+    [MAKEFILE <makefile-name>]
+    [TARGETS <target-name>...]
+    [OPTIONS <make-option>...]
+    [OPTIONS_RELEASE <make-option>...]
+    [OPTIONS_DEBUG <make-option>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### ADD_BIN_TO_PATH
+
+Adds the configure dependent `(debug/)bin` directory to the system path. This is useful if configure builds and runs executables with vcpkg integrated dependencies.
+
+### DISABLE_PARALLEL
+
+Disables parallel building. By default, `make` is called with `-j` to parallelize the build. Use this if the project does not support parallel building.
```

Comment:
"`-j`" might be misunderstood as a literal `-j` without the number of allowed parallel jobs.